tests/pyproject.toml

```diff
@@ -0,0 +1,156 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-dockerimages contributors
```

Comment:
I think I would place the Pyproject.toml file at toplevel. This looks kinda weird.

Reply:
Done. Also applies to the other Python scripts then, which is good.